cirrus/server/cirrus/experiment_recipes.py

```diff
@@ -18,38 +18,62 @@ class RecipeType(Enum):
 
 
 class RemoteSettings:
-    def __init__(self, sdk: SDK):
+    def __init__(self, sdk: SDK, preview_sdk: SDK):
```

Comment:
Okay so the first thing I'll say is rather than having `sdk` and `preview_sdk` I'd suggest we prefix both names so it's more clear which is which, maybe something like `live_sdk` and `preview_sdk`?